src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/client/PrisonsApiClient.kt

```diff
@@ -0,0 +1,22 @@
+package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.client
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.stereotype.Component
+import org.springframework.web.reactive.function.client.WebClient
+import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.response.model.SentenceAndOffenceDetails
+import uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.response.model.SentenceDetail
+
+@Component
+class PrisonsApiClient(
```

Comment:
Let's call this `PrisonApiClient` (remove the `s`) to match the name of [the service we're calling](https://github.com/ministryofjustice/prison-api). And for the `@Qualifier` below.